_posts/2017-04-05-promoting-rust.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "Advocating X effectively, where X is Rust"
+author: Florian Gilcher
+description: "Want to convince people of Rust? Here's a few hints!"
+---
+
+Excited about Rust? Want to talk about it all the time? Then stay a while and listen!
+
+A lot of programming language advocacy falls flat by not being well-prepared, placed wrong or just being plain contra-productive. This is wasted effort on all sides: for the writer to write and for the reader to read - and dismiss.
+
+I hope this article is also useful for people advocating other languages or projects.
+
+## Things to avoid
+
+Let's start by things that you definitely shouldn't do, if you have any interest of convincing someone. It's good to always have these in your head, just to ensure to not do them accidentally.
+
+First of all, don't ask random people to consider Rust or an _existing_ project. They already chose the language they want to use for implementation and as long as they don't come up with a specific problem that brings up the language discussion on again, the topic is through. No one wants to waste more time on topics already discussed.
```

Comment:
Might be worth distinguishing between random people and a project you're involved in.